src/Storage/Database/Schema/Types/JsonType.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Bolt\Storage\Database\Schema\Types;
+
+use Bolt\Common\Exception\DumpException;
+use Bolt\Common\Exception\ParseException;
+use Bolt\Common\Json;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types;
+
+/**
+ * Type generating JSON object values.
+ *
+ * @internal Provided to bridge DBAL 2.5 & 2.6 support.
```

Comment:
Maybe note to drop when 2.5 and PHP 7.0 is dropped?

Reply:
Yep, merged the docblock from the listener method when I moved it over